internal/app/query/query_service_test.go

```diff
@@ -66,3 +70,59 @@ func TestGetPageSize(t *testing.T) {
 		})
 	}
 }
+
+func TestBackfillCache(t *testing.T) {
```

Comment:
Honestly, I'd skip testing here (as I did with tickets), and just make sure all these cases are covered by e2e.

Reply:
Specifically, the "call create and see backfill in query, then call update and the backfill in query is updated" test case on e2e is super important.  Be good to also check with delete properly making it go away.  (Do we have that test case for tickets?  If not we should...)

Doing those tests would make these tests redudant.